daemon/cmd/debuginfo.go

```diff
@@ -48,14 +51,14 @@ func (h *getDebugInfo) Handle(params restapi.GetDebuginfoParams) middleware.Resp
 	dr.CiliumMemoryMap = memoryMap(os.Getpid())
 
 	dr.EnvironmentVariables = []string{}
-	// The debuginfo handler relies on the global Viper instance to list
-	// the agent settings and their values. Thus, this is temporarily commented
-	// out and will be fixed in a subsequent commit.
-	// for _, k := range Vp.AllKeys() {
-	// 	// Assuming we are only getting strings
-	// 	v := fmt.Sprintf("%s:%s", k, Vp.GetString(k))
-	// 	dr.EnvironmentVariables = append(dr.EnvironmentVariables, v)
-	// }
+	var sb strings.Builder
+	for k, v := range d.settings {
+		sb.Reset()
+		sb.WriteString(k)
+		sb.WriteRune(':')
+		sb.WriteString(v)
+		dr.EnvironmentVariables = append(dr.EnvironmentVariables, sb.String())
```

Comment:
Usually I go for the `strings.Builder` to avoid the extra allocation for each `+`. But I guess for this simple case `+` might be faster (and is also more readable). I'm gonna change it. :+1: